CHANGELOG.adoc

```diff
@@ -0,0 +1,223 @@
+= Revision History
```

Comment:
@budhrg the reason for the internal name change is that these types of files can be used as prep files for release notes, because they indicate the changes that were made with each software release. This way, we can automate publishing this info from the file to the doc site in the future.